src/nextjs/package.json

```diff
@@ -9,13 +9,13 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@line/liff": "2.22.2",
-    "next": "11.1.3",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@line/liff": "2.24.0",
```

Comment:
Although not related to the main purpose of this PR, I have changed to use the latest version.